app/controllers/mechanic_rides_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MechanicRidesController < ApplicationController
```

Comment:
Great controller organization! Well done making a RESTful controller action to create this joins record